packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -400,6 +408,16 @@ export const scheduleMicrotask: any =
           .catch(handleErrorInNextTick)
     : scheduleTimeout; // TODO: Determine the best fallback here.
 
+// -------------------
+//     requestAnimationFrame
+// -------------------
+export const supportsAnimationFrame =
+  typeof requestAnimationFrame === 'function';
+export const scheduleAnimationFrame: any =
+  typeof requestAnimationFrame === 'function'
+    ? requestAnimationFrame
+    : undefined;
```

Comment:
Should we fallback for this? 

I also just realized we need to capture it earlier above like we do for `setTimeout` and `Promise`.

Reply:
The fallback behavior should be to act like a normal default update (i.e. the same behavior as today).

We don't need a fallback for requestAnimationFrame because if it's not defined then we won't call it — schedule unknown updates like regular default updates, like we do today.